tfjs-backend-webgpu/src/device_limits.ts

```diff
@@ -15,12 +15,9 @@
  * =============================================================================
  */
 
-export const MAX_COMPUTE_WORKGROUP_INVOCATIONS = 256;
+export let maxComputeWorkgroupsPerDimension: number;
```

Comment:
keep the initial value? `export let maxComputeWorkgroupsPerDimension = 256;`

Reply:
Done